socket.c

```diff
@@ -117,7 +151,7 @@ void bind_to_telnet_port(char type, int *socketdescriptor)
 		exit(EXIT_FAILURE);
 	}
 
-	logg("Listening on port %i for incoming connections", port);
+	logg("Listening on port %i for incoming %s connections", port, dualstack == true ? "IPv4 + IPv6" : "IPv4");
```

Comment:
The `== true` can be removed since `dualstack` is a boolean already.

Reply:
I'm not sure how to detect if the system has a working `ipv6` interface. If we cannot find out how to do it, we could also change the error reporting (don't report on the Individual failures, but only once at the end when we couldn't find at least one port).